_data/2018-19/manual_resources.yaml

```diff
@@ -0,0 +1,33 @@
+budget_speech:
```

Comment:
Why is _data added to this repo? You shouldn't need it for the about page.

In fact, _data is generated from yaml views in this project. In future changes we will make the jekyll pages that depend on _data data use the django views for their context.

In this case, please delete _data.

Reply:
_data includes the videos.yaml about.yaml navbar.yaml, and the about page needs these data now. I used the PyYAML to parse these data to django context